test/e2e/aggregatedapi_test.go

```diff
@@ -0,0 +1,157 @@
+package e2e
+
+import (
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/karmada-io/karmada/test/e2e/framework"
+)
+
+var _ = ginkgo.Describe("aggregatedapi testing", func() {
+	resourceNames := []string{}
+	clustername1 := "member1"
+	clustername2 := "member2"
+	apiPath1 := "/apis/cluster.karmada.io/v1alpha1/clusters/" + clustername1 + "/proxy/api/v1/nodes"
+	apiPath2 := "/apis/cluster.karmada.io/v1alpha1/clusters/" + clustername2 + "/proxy/api/v1/nodes"
+	apisPath1 := "/apis/cluster.karmada.io/v1alpha1/clusters/" + clustername1 + "/proxy/apis"
+	apisPath2 := "/apis/cluster.karmada.io/v1alpha1/clusters/" + clustername2 + "/proxy/apis"
+	proxyclustrole := &rbacv1.ClusterRole{
+		TypeMeta:   metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{Name: "cluster-proxy-clusterrole"},
+		Rules: []rbacv1.PolicyRule{{
+			APIGroups: []string{"cluster.karmada.io"},
+			Verbs:     []string{"*"},
+			Resources: []string{"clusters/proxy"},
+			//ResourceNames: []string{"member1","member2","member3"},
+			ResourceNames: resourceNames,
+		}},
+	}
+
+	member1config := "/root/.kube/members.config"
+	karmadaconfig := "/root/.kube/karmada.config"
```

Comment:
We can use `karmadaClient` to connect `karmada-apiserver`; and use func `GetClusterClient(clusterName string)` to get client of the member cluster.

Reply:
@XiShanYongYe-Chang how can i get the karmada-apiserver Host with karmadaClient
or is there other way i can get the Host

Reply:
We can use:
https://github.com/karmada-io/karmada/blob/0365d04c5dae741cb2517237dda28cc03ead987d/test/e2e/suite_test.go#L68
just like:
https://github.com/karmada-io/karmada/blob/0365d04c5dae741cb2517237dda28cc03ead987d/test/e2e/failover_test.go#L63